### PR TITLE
Fix bug create subs v2

### PR DESCRIPTION
**Pull request by crbrox**
@fgalan 
@kzangeli

### CODE REVIEW

File: test/functionalTest/cases/1316_create_subscription/post_subscriptions.test [hunk 1]
```diff
@@ -74,6 +78,89 @@ echo
 echo
 
 
+echo "03. POST /v2/subscriptions"
```
Comment (fgalan) [inline test/functionalTest/cases/1316_create_subscription/post_subscriptions.test]: Maybe this should mirror the preamble, I mean:

```
echo "03. POST /v2/subscriptions (id)"
```

Similar comment for other steps.

Comment (crbrox) [inline test/functionalTest/cases/1316_create_subscription/post_subscriptions.test]: Fixed 6c6cf7a

**Comment by fgalan** [comment]
LGTM (passing the ball to @kzangeli for final LGTM + merge + delete branch)

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**